### ISSUE TITLE
error max length

### ISSUE THREAD

**Issue by Buwra** [hpfxd/PandaSpigot#194]
![Screenshot_8](https://github.com/hpfxd/PandaSpigot/assets/154678253/b2b6c28f-b901-4888-acde-8ffbf950f82d)

**Comment by Buwra** [comment]
> Probably plugin issue

It's not a plugin problem, I get an error when there is no plugin on the server.

**Comment by andreasdc** [comment]
Do you have bungee?

**Comment by andreasdc** [comment]
Try with default configuration files

**Comment by Buwra** [comment]
> Try with default configuration files

I already tried

**Comment by andreasdc** [comment]
Try with different client version

**Comment by Buwra** [comment]
> Try with different client

There is no connection to different clients, some clients can enter and some cannot, so regarding the code

**Comment by Buwra** [comment]
io.AbstractByteBuf I detect that there is an error.

**Comment by jose27iwnl** [comment]
Try with a new world completly clean.

**Comment by Buwra** [comment]
> Try with a new world completly clean.

I have such a problem with the default fork.

I deleted the world but it still didn't work

**Comment by andreasdc** [comment]
Try with different spigot and try older versions

**Comment by Buwra** [comment]
> Farklı spigot ile deneyin ve eski versiyonları deneyin, sıfırdan yapın, dosya bırakmayın.

Yes, I don't have any problems with different spigots, but I also love this fork.

**Comment by Buwra** [comment]
Can you suggest me a spigot fork like this?

**Comment by andreasdc** [comment]
Try WindSpigot, but it's discontinued so I'm not sure about stability or if it doesn't have any bugs. Some of the latest patches may be risky.

**Comment by uRyanxD** [comment]
Closing this issue because I remember looking for an issue in panda at the time it was created and finding nothing, probably a client issue

**Comment by vire-gh** [comment]
It is the client brand packet, in PacketPlayInCustomPayload packet and the 1.8 NMS packet decoder enforces a hardcoded 20-char limit on certain string reads.
